src/downshift.js

```diff
@@ -723,7 +734,10 @@ class Downshift extends Component {
       ...state,
     })
     this.previousResultCount = resultCount
-    setA11yStatus(status)
+    // TODO: make this work.
+    if (!isReactNative()) {
+      setA11yStatus(status)
```

Comment:
It looks like `setA11yStatus` calls a bunch of DOM-specific APIs, which don't exist on React Native. Should we fork the logic and send this a11y status elsewhere?